contents/docs/deploy.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Deploy PostHog
+sidebar: Docs
+---
+
+PostHog offers a number of different deployment options:
+
+-   [PostHog Cloud](#posthog-cloud) - Create a new PostHog Cloud deployment with just a few clicks.
+    -   Recommended as it's **the fastest way** to try PostHog out.
+    -   Comes with **extra features for reliability and security** and automatic updates.
+-   [Hobby](#hobby-deployment) - Recommended for testing or very small hobby projects _(&lt;100k events/month)_
```

Comment:
I'd swap these two (I'd put self hosted ahead of hobby) since self hosted is the option in line with ideal customer profile